drivers/overlay/overlay.go

```diff
@@ -174,6 +175,18 @@ func Init(home string, options []string, uidMaps, gidMaps []idtools.IDMap) (grap
 		}
 	}
 
+	usingMetacopy, err := doesMetacopy(home, opts.mountOptions)
+	if err == nil {
+		if usingMetacopy {
+			logrus.Debugf("overlay test mount indicated that metacopy is being used")
+		} else {
+			logrus.Debugf("overlay test mount indicated that metacopy is not being used")
+		}
+	} else {
+		logrus.Debugf("overlay test mount did not indicate that metacopy is being used: %v", err)
+		usingMetacopy = false
```

Comment:
This is strange. If you get err returned from doesMetacopy(), then it should be returned back to higher layers?

Reply:
Oh, right, we suppress an ENODATA in `pkg/system`.  Yeah, no harm in reporting it, then.  Changing it.